packages/twenty-front/src/modules/activities/components/ActivityBodyEditor.tsx

```diff
@@ -56,6 +57,28 @@ export const ActivityBodyEditor = ({
     slashMenuItems = slashMenuItems.filter((x) => x.name != 'Image');
   }
 
+  const [uploadFile] = useUploadFileMutation();
+
+  const handleUploadAttachment = async (file: File): Promise<string> => {
+    if (!file) {
+      return '';
+    }
+    const result = await uploadFile({
+      variables: {
+        file,
+        fileFolder: FileFolder.Attachment,
+      },
+    });
+    if (!result?.data?.uploadFile) {
+      throw new Error("Couldn't upload Image");
+    }
+    const imageUrl =
+      process.env.REACT_APP_SERVER_BASE_URL +
```

Comment:
we should use: import { REACT_APP_SERVER_BASE_URL } from '~/config';
This allow run-time environment variable injection